x-pack/plugins/ml/common/util/job_utils.js

```diff
@@ -510,3 +513,32 @@ export function validateGroupNames(job) {
     find: id => (messages.find(m => id === m.id)),
   };
 }
+
+export function normalizeTimes(jobs) {
+  const min = Math.min(...jobs.map(job => +job.timeRange.from));
+  const max = Math.max(...jobs.map(job => +job.timeRange.to));
+
+  const ganttScale = d3.scale.linear().domain([min, max]).range([1, 299]);
+
+  jobs.forEach(job => {
+    if (job.timeRange.to !== undefined && job.timeRange.from !== undefined) {
+      job.timeRange.fromPx = ganttScale(job.timeRange.from);
+      job.timeRange.toPx = ganttScale(job.timeRange.to);
+      job.timeRange.widthPx = job.timeRange.toPx - job.timeRange.fromPx;
+
+      job.timeRange.toMoment = moment(job.timeRange.to);
+      job.timeRange.fromMoment = moment(job.timeRange.from);
+
+      const fromString = job.timeRange.fromMoment.format('MMM Do YYYY, HH:mm');
+      const toString = job.timeRange.toMoment.format('MMM Do YYYY, HH:mm');
+      job.timeRange.label = i18n.translate('xpack.ml.jobSelector.jobTimeRangeLabel', {
```

Comment:
The time range label that is used in the tooltip for the Gantt bar doesn't seem to be displayed according to the Kibana timezone setting.

Reply:
Nice catch! Updated in: 41ba083db291184cdd01d9f27ebf17b9d2688e5f